### DISCUSSION TITLE
Matrix configured (open source Discord/Slack)

### DISCUSSION THREAD

**Discussion by spacecowboy** [spacecowboy/Feeder#240] [Announcements]
I've created a Matrix space for Feeder and here is the invite link: https://matrix.to/#/#feeder:matrix.org

The rooms can also be viewed anonymously without an account, see

* https://app.element.io/#/room/#feeder-support:matrix.org
* https://app.element.io/#/room/#feeder-general:matrix.org

Let's see if it gets any use.

In case you don't know, Matrix can be thought of as an open source alternative to Discord or Slack.

**Comment by MrCamel999** [comment]
Awesome! I will join the room sometime next week when I can access my computer.
